modules/unbound/charts.go

```diff
@@ -182,6 +185,17 @@ var (
 			{ID: "total.num.prefetch", Name: "prefetches"},
 		},
 	}
+	expiredChart = Chart{
+		ID:       "cache_expired",
+		Title:    "Cache Expiries",
```

Comment:
lets make Title `Replies Served From Expired Cache` and units `replies`

Reply:
requested changes applied with c713836727bea7665e419d878e25a3903805fc6d